framework/src/modules/nft/internal_method.ts

```diff
@@ -82,20 +106,95 @@ export class InternalMethod extends BaseMethod {
 		});
 	}
 
-	public async transferInternal(
+	public async verifyTransfer(
```

Comment:
The [checks in LIP](verifyTransferInternal) reflect the newly introduced functions `getNFT`, `isNFTEscrowed` and `isNFTLocked`, and the ordering of checks is different. I believe this will be updated in the SDK PRs that introduce those functions, since you split the task? cc @Incede

Reply:
`getNFT` `isNFTEscrowed` and `isNFTLocked` are now already implemented in [PR](https://github.com/LiskHQ/lisk-sdk/pull/9034) and can be used by rebasing on [this](https://github.com/LiskHQ/lisk-sdk/tree/8959-method-getnft) branch